packages/table/src/Cell/InternalCell.tsx

```diff
@@ -30,6 +30,14 @@ const InternalCell = ({
   const { table, disableAnimations } = useTableContext();
   const isSelectable = !!table && !!table.hasSelectableRows;
   const transitionRef = useRef<HTMLElement | null>(null);
+  const contentRef = useRef<HTMLDivElement>(null);
+
+  const contentHeight = useMemo(
+    () => (contentRef.current ? contentRef.current.clientHeight : 0),
+    // Lint flags `content` as an unnecessary dependency, but we want to update `contentHeight` when the value of `children` changes
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [children],
+  );
```

Comment:
Non-blocking, but can we add a TODO to:
1. break this out into a reusable hook, and
2. Use a `resize` event listener on `contentRef` element to update the height, instead of listening for changes in `children` (which could be flaky)